recipes/semantikon/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "semantikon" %}
+{% set version = "0.0.4" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 98b22a3732aed291a3dfdfb0842e05c27cc3299169313c073ef78faf5e2953f8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+    - versioneer
+  run:
+    - python >={{ python_min }}
+    - numpy >=2.0.0
```

Comment:
```suggestion
    - numpy >=1.26.4, <=2.2.0
```

Reply:
This would allow `pyiron_atomistics` to be installed alongside semanticon (required to be included in the main cmti env :))